tests/v2/validation/provisioning/k3s/provisioning_node_driver.go

```diff
@@ -77,4 +78,12 @@ func TestProvisioningK3SCluster(t *testing.T, client *rancher.Client, provider P
 	podResults, podErrors := pods.StatusPods(client, clusterIDName)
 	assert.NotEmpty(t, podResults)
 	assert.Empty(t, podErrors)
+
+	if psaEnabled {
+		err = clusters.CheckPSACT(client, clusterName)
```

Comment:
there isn't any validation done here when checking whether PSACT is set or not. The checkPSACT function may be a little bit of overkill, since this could be a one liner. 
```
if psaEnabled {
   Is there a case where PSA is enabled, but not having a psact policy? If not, we should just require cluster.DefaultPodSecurityAdmissionConfigurationTemplateName != ""
 otherwise, I still think there should be some require() line here, instead of just logging the psact if it is there. similar to the prime comments earlier this week. 
}

Reply:
I can do that like how we did for the prime checks, that's a fair point.

Reply:
Thinking further on this, wouldn't it make sense to keep the `CheckPSACT`? Even though we're just logging the message, we're pretty much doing that with the serviceaccounttokensecret as well.

Additionally, multiple files have this check, so wouldn't it just be easier to reference this function?

Reply:
Resolved.